.github/CODEOWNERS

```diff
@@ -1,9 +1,8 @@
 # Learn how to add code owners here:
 # https://help.github.com/en/articles/about-code-owners
 
+/crates @vercel/web-tooling
+/Cargo.lock @vercel/web-tooling
+/Cargo.toml @vercel/web-tooling
+/rust-toolchain @vercel/web-tooling
 * @vercel/turbo-oss
-/crates
-/Cargo.lock
-/Cargo.toml
-/rust-toolchain
-/pnpm-lock.yaml
```

Comment:
can we just leave that as nothing?

Reply:
Hey just saw this comment! What do you mean leave that as nothing? As in, nobody owns it?

Reply:
yeah, the lock file could be changed because of something in the `docs` or something in the `crates` folder, so I feel like it alone shouldn't assign anyone 🤷

Reply:
Gotcha. (1) I didn't realize that leaving it empty would assign _no one_ (that's cool if true!), but (2) the way it's structured now is that all files are co-owned by both teams unless specified otherwise. If we left it blank and no-one is assigned, would the PR author pick people to ✅  manually? I think we'd still need approvers, right?